src/SIL.XForge.Scripture/Services/SFProjectService.cs

```diff
@@ -415,7 +413,8 @@ await RealtimeService
                         Email = email,
                         Key = _securityService.GenerateKey(),
                         ExpirationTime = expTime,
-                        ProjectRole = role
+                        ProjectRole = role,
+                        ShareLinkType = ShareLinkType.Recipient
```

Comment:
You may want to add a comma at the end of `ShareLinkType = ShareLinkType.Recipient` (i.e. change to `ShareLinkType = ShareLinkType.Recipient,`), so that if a property is added to ShareKey in future, it will not require this line to be modified (and so make that PR cleaner).